Klarna.Rest/Klarna.Rest.Core/Model/Enum/HostedPaymentPageStatus.cs

```diff
@@ -5,11 +5,13 @@
     /// </summary>
     public enum HostedPaymentPageStatus
     {
-        WAITING, 
+        WAITING,
+        BACK,
         IN_PROGRESS,
         COMPLETED,
         CANCELLED,
         FAILED,
-        LEGACY
```

Comment:
LEGACY is legacy? Removing this field will break the backward comaptibility

Reply:
Fixed

Reply:
Added back enum